CHANGELOG.md

```diff
@@ -23,6 +23,7 @@
 
 * [#8146](https://github.com/rubocop-hq/rubocop/pull/8146): Use UTC in RuboCop todo file generation. ([@mauro-oto][])
 * [#8149](https://github.com/rubocop-hq/rubocop/pull/8149): Cop "Metrics/CyclotomicComplexicy" now counts `&.`, `||=`, `&&=` and blocks known to iterate. Default bumped from 6 to 7. ([@marcandre][])
+* [#8037](https://github.com/rubocop-hq/rubocop/pull/8037): Cop "Metrics/AbcSize" now counts &., ||=, &&=, multiple assignments, for, yield, and iterating blocks. Default bumped from 15 to 16.5. ([@marcandre][])
```

Comment:
Put cop names inside backticks instead of double quotes.

Reply:
Good catch, thanks